### PR TITLE
Travis uses qt-5.5

### LINKED ISSUE #2323: Update our Travis configuration to Qt-5.5 or later
https://travis-ci.org/CGAL/cgal/jobs/259271205#L5309

For CGAL-4.12, we want to require Qt-5.5 or later. That is what we test in the testsuite anyway. Only in Travis we test with Qt-5.2.1.

But I want to modernize our code base, and `QTimer::singleShot` is only compatible with lambda expression from Qt-5.5. That is why we must update the Qt5 version in Travis.

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes

This PR makes our travis config use Qt 5.5.

fixes #2323

### CODE REVIEW

**Comment by maxGimeno** [comment]
This is still a work in progress.

**Comment by maxGimeno** [comment]
If that last commit runs OK on Travis, this branch should be ready for integration.

File: .travis.yml [hunk 1]
```diff
@@ -43,19 +43,13 @@ env:
   - PACKAGE='TDS_3 Three Triangulation ' 
   - PACKAGE='Triangulation_2 Triangulation_3 Union_find ' 
   - PACKAGE='Visibility_2 Voronoi_diagram_2 ' 
-  - PACKAGE='Polyhedron_demo'
-
-
-compiler: 
-  - clang-3.6
-  - gcc  
+  - PACKAGE='Polyhedron_demo'
```
Comment (lrineau) [inline .travis.yml]: You have removed the test of clang!

Comment (maxGimeno) [inline .travis.yml]: Indeed, it is still in the template though. There seem to be a problem in the script generate_travis.sh, I'll look into it right away

**Comment by maxGimeno** [comment]
fixed

**Comment by lrineau** [comment]
As it is rather long to test changes to Travis, let's merge it now, and we may experiment other ways later.

**Comment by sloriot** [comment]
@lrineau Shall we backport this PR to 4.11 ?

**Comment by maxGimeno** [comment]
I don't think so, we need it for a PR that was not in 4.11.

2017-08-09 9:38 GMT+02:00 Sebastien Loriot <notifications@github.com>:

**Comment by sloriot** [comment]
I think the PR was https://github.com/CGAL/cgal/pull/2312 and it was merged in 4.11

**Comment by maxGimeno** [comment]
Ok, I don't know why it is in 4.11, but still, I don't think 4.11 has
travis tests.

2017-08-09 10:29 GMT+02:00 Sebastien Loriot <notifications@github.com>:

**Comment by lrineau** [comment]
All branches and all the pull-requests with a `.travis.yml` file have Travis tests. That is since CGAL-4.9.

https://travis-ci.org/CGAL/cgal/branches

**Comment by lrineau** [comment]
I have applied the patch to the branch CGAL-4.11, with the commit https://github.com/CGAL/cgal/commit/7610778af7a8a152a7d41e4521cc02478ce3bfa0.

**Pull request merged**